day_5/Day_5_Final_Project_Password_Generator.py

```diff
@@ -0,0 +1,21 @@
+#Password Generator Project
+import random
+letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+symbols = ['!', '#', '$', '%', '&', '(', ')', '*', '+']
+print("Welcome to the PyPassword Generator!")
+nr_letters= int(input("How many letters would you like in your password?\n"))
+nr_symbols = int(input(f"How many symbols would you like?\n"))
+nr_numbers = int(input(f"How many numbers would you like?\n"))
+#Eazy Level - Order not randomised:
```

Comment:
I can see that you have completed only the "Eazy Level". That is not enough, I know you can do better 😄 . 

Here is what is left to do
- Complete the "Hard Level" task, meaning that the order of the characters in the password should be randomised.
- Complete the * task from me: After you are done with the "Hard Level" task you should have 2 password printed 1) Easy Level 2) Hard Level. When you will do the "Hard Level" you most likely(as Angela shows on the video) will use special random module function. So, my task with * is that I **forbid** you using this special function from the random module. To help you I can say that you have 2 options.
  - After creating a password using the "Easy Level" emulate the behaviour of the special `random` module function.
  - Copy the algorithm of the "Easy Level" and think how it can be changed in order for the password to have randomised order of the characters.